manager/src/main/java/com/redhat/service/smartevents/manager/api/user/validators/processors/ErrorHandlerConstraintValidator.java

```diff
@@ -36,6 +40,10 @@ public boolean isValid(BridgeRequest bridgeRequest, ConstraintValidatorContext c
             return true;
         }
 
+        if (BridgeDefinition.isEndpointErrorHandlerAction(errorHandlerAction)) {
```

Comment:
Same comment as above, I'd suggest to move this as a method onto the `ProcessingErrorService` and leave the `BridgeDefinition` untouched. I think it helps with encapsulating the changes.

Reply:
done :+1: